src/libraries/Microsoft.PowerFx.Core/Types/LazyTypeProvider.cs

```diff
@@ -35,6 +33,11 @@ public LazyTypeProvider(AggregateType type)
             BackingFormulaType = type;
         }
 
+        public LazyTypeProvider(TableParameters tableParameters)
+        {
+            BackingFormulaType = tableParameters?.RecordType as AggregateType;
+        }
```

Comment:
Does this mean now we can create LazyTypeProvider without any BackingFormulaType? Ie null

Reply:
I don't understand what you mean here
tableParameters can be null (for non-delegable items or tests)

Reply:
Nope